app/config/collections/common.php

```diff
@@ -364,6 +364,94 @@
                 'array' => false,
                 'filters' => ['datetime'],
             ],
+            [
+                '$id' => ID::custom('planId'),
```

Comment:
Stripe vendor locking here again. Also, we don't want to couple plans with specific users, we need to fine a solution to be able to use them for both users or teams. I would avoid changing teams and users collections and rely one the new collections to create the link, this link is probably a `subscriptions` collection.